compose/src/main/java/com/bottlerocketstudios/compose/home/HomeCard.kt

```diff
@@ -88,6 +97,63 @@ fun CardLayout(userRepositoryUiModel: UserRepositoryUiModel, selectItem: (userRe
     }
 }
 
+@Composable
+fun PullRequestCardLayout(userPullRequestUIModel: UserPullRequestUIModel) {
+    userPullRequestUIModel.pullRequest.title
+    Card(
+        elevation = Dimens.plane_3,
+        modifier = Modifier.fillMaxWidth()
+    ) {
+        Row(modifier = Modifier.padding(top = Dimens.grid_2, bottom = Dimens.grid_2)) {
+            @Suppress("MagicNumber")
+            Column(
+                modifier = Modifier.padding(start = Dimens.grid_2_5, end = Dimens.grid_2_5)
+            ) {
+                Row {
+                    ResponsiveText(
+                        text = userPullRequestUIModel.pullRequest.title,
+                        textStyle = typography.h3,
+                        modifier = Modifier
+                            .padding(end = Dimens.grid_1)
+                            .weight(3f)
+                    )
+                    Text(
+                        text = userPullRequestUIModel.formattedLastUpdatedTime.getString(),
+                        fontSize = 10.sp
+                    )
+                }
+                Row(
+                    modifier = Modifier.padding(top = Dimens.grid_0_5),
+                    horizontalArrangement = Arrangement.spacedBy(Dimens.grid_2)
+                ) {
+                    Text(
+                        text = userPullRequestUIModel.pullRequest.source,
+                        style = typography.body1,
+                        color = Colors.onBackground,
+                        modifier = Modifier
+                            .background(color = brown_grey, shape = RoundedCornerShape(Dimens.grid_0_5))
+                            .padding(Dimens.grid_0_75)
+                    )
+                    Text(text = "\u2192", style = typography.body1)
```

Comment:
@LuisRocketRangel No fault of your own but I did want to mention some recent changes I've made :)

Note that I've replaced the rendered unicode right arrow character with the numeric representation. This **should** have been caught when running the Android Studio `lintInternalDebug` task locally **but wasn't because the BR custom android lint rules weren't added to that module** (until now).

Overall, I just pushed a review commit that adds the custom lint rules to the compose module and replaced the rendered unicode characters with the numeric representations.

More info on why those rendered unicode characters aren't allowed in source code at https://github.com/BottleRocketStudios/Android-CustomLintRules (integrated into the arch demo in https://github.com/BottleRocketStudios/Android-ArchitectureDemo/pull/41)